snippets/graphql/get-scorecard/README.md

```diff
@@ -0,0 +1,53 @@
+# How to use this
+
+This query will retrieve all specified information for a given scorecard.
+
+Replace `scorecardARI` below in the variables section with a valid Compass component ARI in your site and execute the query. You can use [the GraphQL explorer](https://developer.atlassian.com/cloud/compass/graphql/explorer/) to run this query and explore [the Compass API](https://developer.atlassian.com/cloud/compass/graphql/) further.
+
+### Query
+
+```graphql
+query getScorecard($scorecardARI: ID!) {
+  compass {
+    scorecard(id: $scorecardARI) {
+      __typename
+      ... on CompassScorecard {
+        id
+        name
+        description
+        ownerId
```

Comment:
I got an error when running this query in the explorer. Does the latest version still use `ownerId`? It worked with the following, though: 

```
owner {
    id
}
```